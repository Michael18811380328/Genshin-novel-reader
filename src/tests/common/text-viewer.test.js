import renderer from 'react-test-renderer';
import TextViewer from '../../common/text-viewer';

it('render text-viewer', () => {
  const context = 'This is test txt file.';
  const component = renderer.create(
    <div>
      <TextViewer context={context}/>
    </div>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
