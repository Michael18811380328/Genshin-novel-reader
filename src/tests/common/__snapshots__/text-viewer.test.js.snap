// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render text-viewer 1`] = `
<div>
  <div>
    <p>
      This is test txt file.
    </p>
  </div>
</div>
`;
